server/src/main/java/org/apache/iotdb/db/engine/merge/manage/MergeManager.java

```diff
@@ -74,13 +75,18 @@
   private MergeManager() {
   }
 
-  public RateLimiter getMergeRateLimiter() {
-    setMergeRate(IoTDBDescriptor.getInstance().getConfig().getMergeThroughputMbPerSec());
-    return mergeRateLimiter;
+  public RateLimiter getMergeWriteRateLimiter() {
+    setWriteMergeRate(IoTDBDescriptor.getInstance().getConfig().getMergeWriteThroughputMbPerSec());
```

Comment:
every time using `getMergeWriteRateLimiter()` will call this set method `setWriteMergeRate `,if this para can be changed after IoTDB start, this is OK, otherwise...

Reply:
It may can be set afterward in cli